api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/staticapi/StaticAPIRefreshController.java

```diff
@@ -16,20 +16,22 @@
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
 
 @RestController
-@RequestMapping("/static-api")
+@RequestMapping({"/apicatalog/static-api", "/apicatalog/api/v1/static-api"})
 @RequiredArgsConstructor
 public class StaticAPIRefreshController {
 
     private final StaticAPIService staticAPIService;
 
     @PostMapping(value = "/refresh", produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<String> refreshStaticApis() {
-        StaticAPIResponse staticAPIResponse = staticAPIService.refresh();
-        return ResponseEntity
-            .status(staticAPIResponse.getStatusCode())
-            .body(staticAPIResponse.getBody());
+    public Mono<ResponseEntity<String>> refreshStaticApis() {
+        return Mono.fromCallable(staticAPIService::refresh)
```

Comment:
not asking for a change here, but we should be careful when calling this code if it's not 100% sure all paths are using non-blocking IO
To be sure it will work we can rather use publishOn and use one of the provided thread pools that can run blocking IO safely.

Reply:
I have added subscribeOn